taichi/ui/common/renderable_info.h

```diff
@@ -7,7 +9,8 @@ TI_UI_NAMESPACE_BEGIN
 struct RenderableInfo {
   FieldInfo vbo;
   FieldInfo indices;
-  bool has_per_vertex_color;
+  bool has_per_vertex_color{false};
```

Comment:
I'm guessing this property could be removed in the future, as `vbo_attrs` should be enough to capture this information.
But fine for now.